bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/discovery/ShellyThingCreator.java

```diff
@@ -457,10 +457,9 @@ public class ShellyThingCreator {
             Map.entry(SHELLYDT_PRO1PM, THING_TYPE_SHELLYPRO1PM), //
             Map.entry(SHELLYDT_PRO1PM_2, THING_TYPE_SHELLYPRO1PM),
             Map.entry(SHELLYDT_PRO1PM_3, THING_TYPE_SHELLYPRO1PM), //
-            Map.entry(SHELLYDT_PRO1CB, THING_TYPE_SHELLYPRO1CB),
-            Map.entry(SHELLYDT_PRO2_RELAY, THING_TYPE_SHELLYPRO2_RELAY),
-            Map.entry(SHELLYDT_PRO2_RELAY_2, THING_TYPE_SHELLYPRO2_RELAY),
-            Map.entry(SHELLYDT_PRO2_RELAY_3, THING_TYPE_SHELLYPRO2_RELAY),
+            Map.entry(SHELLYDT_PRO1CB, THING_TYPE_SHELLYPRO1CB), Map.entry(SHELLYDT_PRO2, THING_TYPE_SHELLYPRO2_RELAY),
```

Comment:
Are these changes related to firmware 1.6.x?

```suggestion
            Map.entry(SHELLYDT_PRO1CB, THING_TYPE_SHELLYPRO1CB), //
            Map.entry(SHELLYDT_PRO2, THING_TYPE_SHELLYPRO2_RELAY),
```